typing/env.ml

```diff
@@ -27,16 +27,17 @@ module String = Misc.Stdlib.String
 
 let add_delayed_check_forward = ref (fun _ -> assert false)
 
-let value_declarations : ((string * Location.t), (unit -> unit)) Hashtbl.t =
-  Hashtbl.create 16
-    (* This table is used to usage of value declarations.  A declaration is
-       identified with its name and location.  The callback attached to a
-       declaration is called whenever the value is used explicitly
-       (lookup_value) or implicitly (inclusion test between signatures,
-       cf Includemod.value_descriptions). *)
-
-let type_declarations = Hashtbl.create 16
-let module_declarations = Hashtbl.create 16
+type 'a usage_tbl = ('a -> unit) Types.Uid.Tbl.t
+(** This table is used to usage of value declarations.
```

Comment:
Could you take the opportunity to fix the broken sentence?

Reply:
Done.